VAMobile/src/constants/common.ts

```diff
@@ -48,4 +48,12 @@ export const MAX_DIGITS_AFTER_FORMAT = 14
 export const EMAIL_REGEX_EXP = new RegExp(
   /^(([^<>()\]\\.,;:\s@"]+(\.[^<>()\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/,
 )
+export const MAIL_TO_REGEX_EXP = new RegExp(
+  /^(mailto:([^<>()\]\\.,;:\s@"]+(\.[^<>()\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/,
+)
+
+export const PHONE_REGEX_EXP = new RegExp(/^[-0-9()+]{10,}/)
```

Comment:
so this covers anything that is specifically 0-9 ( ) - or + with at least 10 characters which will typically only be phone numbers as it would otherwise be 10s of billions of something else which I think would be exceedingly rare. I could potentially add a cap of it looking at 17 characters to cover +123(456)383-2020.